src/conf.py

```diff
@@ -50,7 +50,7 @@
 
 # General information about the project.
 project = 'Topology Filters - Notes'
-copyright = '2021, Carlos Caralps'
+copyright = '2021, Natalia Castellana'
```

Comment:
```suggestion
copyright ='2020, Natalia Castellana'
```

Et proposo aquest canvi

Reply:
Quedem en 2021